packages/stringsifter/PKGBUILD

```diff
@@ -32,10 +31,10 @@ package() {
 
   install -dm 755 "$pkgdir/usr/bin"
   install -dm 755 "$pkgdir/usr/share/$pkgname"
-  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" README.md *.txt
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" README.md
```

Comment:
Did you try replacing the joinscript by `python setup.py install --root="$pkgdir" --prefix=/usr -O1 --skip-build` ?